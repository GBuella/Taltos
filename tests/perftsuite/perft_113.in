setboard 7k/8/8/1p6/P7/8/8/7K b - - 0 1
echo 7k/8/8/1p6/P7/8/8/7K b - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
