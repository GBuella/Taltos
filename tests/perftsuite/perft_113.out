7k/8/8/1p6/P7/8/8/7K b - - 0 1
5
22
139
877
6112
41874
